### ISSUE TITLE
Google requests "updated browser" for new sign-ins

### ISSUE THREAD

**Issue by David-W-B** [klinker24/google-calendar-desktop#7]
Am using latest version 1.7.1.
I changed my Google password, got logged out everywwhere
Opened calendar, obviously signed out
Tried to log in
Google tells me it wants an updated browser, and so no login possible
![GITHUB Temp](https://user-images.githubusercontent.com/64195502/80081395-2e130680-8553-11ea-88ed-84a5d64846d3.jpg)
![GITHUB Temp 2](https://user-images.githubusercontent.com/64195502/80081702-93ff8e00-8553-11ea-8adb-dbf4b9665ac6.jpg)

**Comment by klinker24** [comment]
Yeah I solved this once before, but seems like Google found a new way to detect Electron. I personally have moved to a Chrome app version and I am not sure if I will solve this issue or not.

**Comment by David-W-B** [comment]
update:
the day after I posted I tried opening it again and worked and is still working... 
¯\_(ツ)_/¯

**Comment by bmitchinson** [comment]
👋 believe I was the one having the issue originally. A fresh install on 10.15.4 with 2FA on my google account worked fine. Just got recommended to use a "new" browser.

@klinker24 "moved to a chrome app version"? curious about it, mind sharing? need a lightweight email client for multiple inboxes as well.

**Comment by kingkool68** [comment]
@bmitchinson See https://www.getmailflow.com/post/gmail-desktop-app

**Comment by David-W-B** [comment]
Hi @klinker24 I'm sure you know but FYI it's happening again - Google requests an updated browser. Have uninstalled and tries reinstalling several times, rebooting, letting days go by, no change.

**Comment by Gr-Nobel** [comment]
is this the end of it?

**Comment by LiuZhenyan-Wuzhong** [comment]
Hello, dear author. Can you recommand a specificial version of chrome browser? I have come across the same problem and haven't solved it.
